package.json

```diff
@@ -28,7 +28,7 @@
 
     "bootstrap": "lerna bootstrap && lerna link",
     "lint": "lerna run lint",
-    "test": "lerna run teststore && lerna run --parallel test",
+    "test": "lerna run --parallel test && lerna run teststore",
```

Comment:
I think it's better to call `teststore` from `neuron-wallet` package instead.

Reply:
ok